app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+var lastValue = null;
+
+const getVersion = gql`
+  {
+    schemaSha
+  }
+`;
+
+export const Version = () => (
+  <Query query={getVersion}>
+    {({ loading, error, data }) => {
+      if (loading) {
+        return <LoadingMessage />;
+      }
+
+      if (error) {
+        return <ErrorMessage />;
+      }
+
+      var newValue = data.schemaSha;
+      if (lastValue == null) {
+        lastValue = newValue;
+      } else if (newValue !== lastValue) {
+        lastValue = newValue;
+        object.location.reload();
```

Comment:
does this work?
I think you want `window.location.reload()`

Reply:
Yeah it was meant to be `window.location.reload()`. object is not correct.